src/main/java/com/internet/shop/db/Storage.java

```diff
@@ -0,0 +1,21 @@
+package com.internet.shop.db;
+
+import com.internet.shop.models.Order;
+import com.internet.shop.models.Product;
+import com.internet.shop.models.ShoppingCart;
+import com.internet.shop.models.User;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage {
+    public static final List<Product> PRODUCTS = new ArrayList<>();
+    public static final List<Order> ORDERS = new ArrayList<>();
+    public static final List<ShoppingCart> SHOPPING_CARTS = new ArrayList<>();
+    public static final List<User> USERS = new ArrayList<>();
```

Comment:
i think naming must be in lowercase for storages. but not sure. And shoppingCarts <-- like this

Reply:
I'll wait for the mentor's review to assure.
Makes sense to me making the fields final, that's where the naming came from.

Reply:
I would agree with @andriylagun 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names